Assets/Scripts/AI/AIManager.cs

```diff
@@ -1,8 +1,94 @@
 ﻿using UnityEngine;
 using System.Collections;
-
+/*
+stealth mini game triggered when one enemy is in the suspicious state
+one list of enemies
+*/
+ //public function that decrements timer of all ai in the alert/suspicious state
 public class AIManager : MonoBehaviour {
-	Transform[] AI_children;
+
+    public GameObject[] AiChildren;
+    public bool playerHidden;
+    public int numberChasing;
+    public static AIManager instace = null;
+
+    private static AIManager instance;
+
+    private AIManager() { }
+
+    void Awake()
+    {
+        if (instance == null)
+        {
+            instance = this;
+        }
+        else if (instance != this)
+        {
+            Destroy(gameObject);
+        }
+        DontDestroyOnLoad(gameObject);
+    }
+
+
+    void Start ()
+    {
+        AiChildren = new GameObject[transform.childCount];
+        int childCount = 0;
+        foreach (Transform child in transform)
+        {
+            AiChildren[childCount] = child.gameObject;
+            childCount++;
+        }
+    }
+
+
+    public int checkChasing()
```

Comment:
Class methods always always have the letters of every word capitalized, so this method should be named CheckChasing()